Formula/opencv@2.rb

```diff
@@ -67,7 +67,8 @@ def install
     # https://github.com/Homebrew/homebrew-science/issues/2302
     args << "-DCMAKE_PREFIX_PATH=#{py_prefix}"
 
-    args << "-DENABLE_SSE41=ON" << "-DENABLE_SSE42=ON" if MacOS.version.requires_sse42?
+    args << "-DENABLE_SSE41=ON" << "-DENABLE_SSE42=ON" \
+      if Hardware::CPU.intel? && MacOS.version.requires_sse42?
```

Comment:
Did `brew audit` not let you put this in an `if` block?

Reply:
Nope, it didn't like the fact that it was a one-line if block.